models/monai_prostate158/utils/Prostate158Runner.py

```diff
@@ -0,0 +1,85 @@
+"""
+-------------------------------------------------
+MHub - MONAI Prostate158 Runner
+-------------------------------------------------
+
+-------------------------------------------------
+Author: Cosmin Ciausu
+Email:  cciausu97@gmail.com
+-------------------------------------------------
+"""
+# TODO: support multi-i/o and batch processing on multiple instances
+
+from typing import List, Optional
+import os, subprocess, shutil, glob, sys
+import SimpleITK as sitk, numpy as np
+from mhubio.core import Module, Instance, InstanceData, DataType, FileType, IO
+from mhubio.modules.runner.ModelRunner import ModelRunner
+import json
+
+@IO.Config('apply_center_crop', bool, False, the='flag to apply center cropping to input_image')
+
```

Comment:
I suggest we remove all empty lines between decorators and their target. This is a personal preference but it makes the relation more clear imo.